### DISCUSSION TITLE
Notice2 app has multiple problems: not able to produce valid notices, not able to submit a notice to CVS

### DISCUSSION THREAD

**Discussion by alexmohora** [OP-TED/eForms-SDK#333] [Q&A] [answered]
I assume Notice2 app is in production but there is no way to use it in a production environment. Even in preview environment it's not possible to obtain a valid notice even if a user fill in data with caution.

Main issue: ta application is not yet able to validate, at GUI level, the elements that are choices in XSD. For example parameters for award criteria - the user has no clue that it should choose only one code because the GUI suggest it can be multiple choices. Also the Organization indicators for Group Leader, Acquiring CPB Awarding CPB OR Natural Person are set to false - user didn't fill anything in GUI but this list is invalid because is a choice in XSD - please see the pictures.

This is a major issue because the GUI seems not to be able to conform with XSD. Also the app is not able to provide errors regarding XSD validation.

Second major issue: the Validation (submit to CVS) is not working - there is a generic error that says that not all mandatory fields are present but there is no indication which one are missing.

Another major issue: the application is not able to adapt dynamically when user fill in data. For example you can choose that there is no electronic auction but still fill in data describing the electronic auction. This can be easily avoided by user but there are other situations more tricky: contract duration can be filled in as a start date and date but in the same time as a period. This kind of inconsistence I suppose will be detected by CVS but, again, the validation is not working.

These issues should be fixed because this application is a good source of reference and a good one for fill in data and obtain a lot of XMLs for different form types and different situations. It is important for helping the development process of eSenders systems, especially the systems that can't integrate the SDK because of their architecture.   
![Screenshot_43](https://user-images.githubusercontent.com/115160065/218423189-2bd6c17d-7766-48d2-b229-c9c7564ab80c.png)
![Screenshot_42](https://user-images.githubusercontent.com/115160065/218423195-4d68013c-64ec-4a12-934d-934a62bef7f8.png)
![Screenshot_41](https://user-images.githubusercontent.com/115160065/218423199-d97a6d01-c975-4291-b977-e992e444dae9.png)
![Screenshot_40](https://user-images.githubusercontent.com/115160065/218423201-8387b29e-cd5f-4724-a3e9-18bdda33ec80.png)

**Comment by bertrand-lorentz** [comment]
As this is not about a bug in the SDK, I'm moving it to our discussion forum.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Please note that eNotices2 is developed separately. For concerns and questions regarding eNotices2, please get directly in touch with the eNotices 2 helpdesk at ted@publications.europa.eu
KR,
